packages/medusa/oas/admin-spec3-base.yaml

```diff
@@ -231,65 +231,65 @@ tags:
       their sessions.
   - name: App
     description: App endpoints that allow handling apps in Medusa.
-  - name: Batch Job
+  - name: Batch Jobs
```

Comment:
note: for reference https://github.com/medusajs/medusa/blob/2b8c33f08b8761014312ac3bfab379296b8ad538/packages/medusa-js/src/resources/admin/index.ts#L41-L77